modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsManagerImpl.java

```diff
@@ -225,51 +679,58 @@ public IgniteStatisticsRepository statisticsRepository() {
      * @param tblPartStats Collection of all local partition level statistics by specified key.
      * @return Local level aggregated statistics.
      */
-    public ObjectStatisticsImpl aggregateLocalStatistics(StatsKey key, Collection<ObjectPartitionStatisticsImpl> tblPartStats) {
+    public ObjectStatisticsImpl aggregateLocalStatistics(
+            StatsKey key,
+            Collection<ObjectPartitionStatisticsImpl> tblPartStats
+    ) {
         // For now there can be only tables
-        GridH2Table table = schemaMgr.dataTable(key.schema(), key.obj());
+        GridH2Table tbl = schemaMgr.dataTable(key.schema(), key.obj());
 
-        if (table == null) {
+        if (tbl == null) {
             // remove all loaded statistics.
-            log.info("Removing statistics for object " + key + " cause table doesn't exists.");
+            if (log.isDebugEnabled())
+                log.debug(String.format("Removing statistics for object %s.%s cause table doesn't exists.",
+                        key.schema(), key.obj()));
+
             statsRepos.clearLocalPartitionsStatistics(key);
         }
-        return aggregateLocalStatistics(table, table.getColumns(), tblPartStats);
+        return aggregateLocalStatistics(tbl, tbl.getColumns(), tblPartStats);
     }
 
     /**
      * Aggregate partition level statistics to local level one.
      *
      * @param tbl Table to aggregate statistics by.
-     * @param selectedColumns Columns to aggregate statistics by.
+     * @param selectedCols Columns to aggregate statistics by.
      * @param tblPartStats Collection of partition level statistics.
      * @return Local level statistics.
      */
     private ObjectStatisticsImpl aggregateLocalStatistics(
             GridH2Table tbl,
-            Column[] selectedColumns,
+            Column[] selectedCols,
             Collection<ObjectPartitionStatisticsImpl> tblPartStats
     ) {
```

Comment:
Let's remove redundant method.

Reply:
Sure, I'll clean the class. But this particular method isn't redundant as fas as I see.